crawler/utils/dockerutils.py

```diff
@@ -360,3 +363,31 @@ def get_docker_container_rootfs_path(long_id, inspect=None):
         raise DockerutilsException('Not supported docker storage driver.')
 
     return rootfs_path
+
+
+def poll_container_create_events(timeout=0.1):
+    try:
+        client = docker.Client(base_url='unix://var/run/docker.sock',
+                               version='auto')
+        filters = dict()
+        filters['type'] = 'container'
+        filters['event'] = 'start'
+        events = client.events(filters=filters, decode=True)
+        with Timeout(seconds=timeout):
+            # we are expecting a single event
```

Comment:
The function returns a generator, don't understand why the restriction on one single event.